client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -341,7 +342,7 @@ class TransferDomainPrecheck extends React.Component {
 		const { currentStep } = this.state;
 		// We disallow HEs to submit the transfer
 		const disableButton =
-			false === unlocked || ! authCodeValid || currentStep < 3 || isSupportUserSession();
+			false === unlocked || ! authCodeValid || currentStep < 3 || this.props.isSupportSession;
```

Comment:
This is an important case. Thanks for catching it.

Reply:
why add this here but not in line 340 above? is there a reason to treat `isSupportSession` differently than `authCodeValid` and `unlocked` and `currentStep`?

Reply:
answer: ESLint was warning about shadowing